addons/binding/org.openhab.binding.pulseaudio/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pulseaudio"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <channel-type id="volume">
+        <item-type>Dimmer</item-type>
+        <label>Volume</label>
+        <description>Volume of an audio item in percent</description>
+        <category>SoundVolume</category>
+        <tags>
+      		<tag>volume</tag>
+    	</tags>
+    	<state min="0" max="100" step="1" pattern="%d" readOnly="false">
+        </state>
+    </channel-type>
+    <channel-type id="mute">
+		<item-type>Switch</item-type>
+		<label>Mute</label>
+		<description>Mute an device</description>
+		<tags>
+      		<tag>mute</tag>
+    	</tags>
+	</channel-type>
+	<channel-type id="state">
+		<item-type>String</item-type>
+		<label>State</label>
+		<description>Current state of a device</description>
+		<tags>
+      		<tag>state</tag>
+    	</tags>
+    	<state readOnly="true">
+    		<options>
+    			<option value="SUSPENDED">Suspended</option>
+    			<option value="IDLE">Idle</option>
+    			<option value="RUNNING">Running</option>
+    			<option value="CORKED">Corked</option>
+    			<option value="DRAINED">Drained</option>
+    		</options>
+    	</state>
+	</channel-type>
+	<channel-type id="slaves">
+		<item-type>String</item-type>
+		<label>Slaves</label>
+		<description>Slave sinks of a combined sink</description>
+		<category>String</category>
```

Comment:
"String" is not a valid category - please see https://github.com/eclipse/smarthome/blob/master/docs/sources/architecture/thing-definition.md#channel-categories